make-all.sh

```diff
@@ -146,5 +146,16 @@ cd bench/yaxpeax
 $make
 cd ../..
 
-echo "[+] All done!"
+echo "[*] Building sleigh benchmark ..."
+cd bench/sleigh
+cmake -DCMAKE_BUILD_TYPE=Release .
+if [[ "$is_windows" == "y" ]]; then
+    msbuild.exe bench-sleigh.sln -p:Configuration=Release -p:Platform=x64
+    cp Release/*.exe .
+    cp sleigh/Release/Sleigh.dll .
+else
+    $make -j4
+fi
```

Comment:
Maybe this to remove the `msbuild.exe` call
```suggestion
cmake --build build --config Release -j 4
if [[ "$is_windows" == "y" ]]; then
    cp build/Release/*.exe .
    cp build/sleigh/Release/Sleigh.dll .
else
    cp build/bench-sleigh-fmt .
fi
```

Reply:
Same as above.